src/core/Akka/Actor/Internal/ActorSystemImpl.cs

```diff
@@ -132,8 +132,16 @@ public override ActorSelection ActorSelection(string actorPath)
 
         private void ConfigureScheduler()
         {
-            var schedulerType = Type.GetType(_settings.SchedulerClass, true);
-            _scheduler = (IScheduler) Activator.CreateInstance(schedulerType, this);
+            try
+            {
+                var schedulerType = Type.GetType(_settings.SchedulerClass, true);
+                _scheduler = (IScheduler)Activator.CreateInstance(schedulerType, this);
```

Comment:
We need to check that the type is an IScheduler or throw configuration exception